cpp/serve/model.h

```diff
@@ -103,6 +160,17 @@ class ModelObj : public Object {
   virtual NDArray BatchPrefill(const ObjectRef& embeddings, const std::vector<int64_t>& seq_ids,
                                const std::vector<int>& lengths) = 0;
 
+  /*!
+   * \brief Batch prefill function. Hidden_states in, hidden_states out.
```

Comment:
Maybe we need to explain the difference of the input hidden states and output hidden states, or otherwise it looks confusing. Same for `BatchDecodeToLastHidden` and `BatchVerifyToLastHidden`.